app/helpers/application_helper.rb

```diff
@@ -1,2 +1,11 @@
 module ApplicationHelper
+  def message(flash)
+    flash.each do |type, msg|
+      if type == "notice"
+        return haml_tag :p, msg, class: "alert alert alert-success", role: "alert"
+      else type == "alert"
+        return haml_tag :p, msg, class: "alert alert alert-danger", role: "alert"
```

Comment:
we don't need to explicitly use `return` - Ruby automatically returns the last line executed in the statement. Moreover - do you think we need this `else` here? Is there any other case than `notice` or `alert`?